### ISSUE TITLE
Make NoCode application backed by Blockchain technology

### ISSUE THREAD

**Issue by thuansoldier** [kelseyhightower/nocode#2258]
Make NoCode application backed by Blockchain technology

**Comment by lanskey** [comment]
+1

**Comment by jbreckmckye** [comment]
+0

**Comment by maxpou** [comment]
It's already supported: see first release (https://github.com/kelseyhightower/nocode/releases/tag/1.0.0)

![](https://user-images.githubusercontent.com/6811830/35903516-6618323e-0b94-11e8-9705-1118e9522bd6.png)

**Comment by tarikhagustia** [comment]
@thuansoldier yes you can use block chain tech, with this repo, here example to run Blockchain support 
```

```

hope this help ..
